### ISSUE TITLE
Incorrect example copy font sizes

### ISSUE THREAD

**Issue by missmatsuko** [Pushedskydiver/Colour-Contrast-Checker#20]
Re: the font sizes at the bottom of [the Colour Contrast Checker web page](https://colourcontrast.cc/):

<img width="1307" alt="Screenshot of example copy." src="https://user-images.githubusercontent.com/17884198/103444900-411dc080-4c22-11eb-9b7e-bf12ac12047a.png">

The headings are "large text - 14pt" and "normal copy - 16px". 

[WCAG 2.1 defines "large text" as "at least 18 point or 14 point bold"](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html#dfn-large-scale). The Understanding Success Criterion page further states that "The ratio between sizes in points and CSS pixels is 1pt = 1.333px, therefore 14pt and 18pt are equivalent to approximately 18.5px and 24px". Since the large text's example copy is not bold, its font size should be 18pt/24px, not 14pt/18.5px.

**Comment by Pushedskydiver** [comment]
Thank you for spotting this. I shall get this updated.

**Comment by Pushedskydiver** [comment]
Should be sorted now. Apologies for how long this has taken me to sort but I am finally in a position where I can put some dedicated time to this tool 😁
